components/org.wso2.carbon.identity.api.server.flow.management/org.wso2.carbon.identity.api.server.flow.management.v1/src/main/resources/flow.yaml

```
@@ -206,6 +206,282 @@ paths:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+        '403':
+          description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
+
+  /flow/in-flow-extension/context-tree:
```

YAMLlint (1.38.0)

[error] duplication of key "/flow/in-flow-extension/context-tree" in mapping

(key-duplicates)